Cargo.toml

```diff
@@ -405,7 +389,7 @@ solana-account-decoder = { path = "account-decoder", version = "=4.0.0-alpha.0",
 solana-account-decoder-client-types = { path = "account-decoder-client-types", version = "=4.0.0-alpha.0", features = ["agave-unstable-api"] }
 solana-account-info = "3.0.0"
 solana-accounts-db = { path = "accounts-db", version = "=4.0.0-alpha.0", features = ["agave-unstable-api"] }
-solana-address = "1.0.0"
+solana-address = "1.1.0"
```

Comment:
semver trick to workaround the issue of multiple versions of Solana-address.

Reply:
Can you explain why are we bumping it?

Reply:
This is a recommendation from @joncinque .This is because the solanin-packet 4.0.0 depends on solana-address v 2.0.0. And other crates are using solana-address v1.0.0. The discrepancy cause compilation error of solana-address::Address (v.2.0.0) is incompatible with olana-address::Address  (v1.0.0)

From Jon:

v1.1 of solana-address re-exports v2: https://github.com/anza-xyz/solana-sdk/blob/265bca4715bc1975bf7c6d012c8dcb13545f971c/address/Cargo.toml#L47 so both types are compatible

we used the semver-trick between the two major versions: https://github.com/dtolnay/semver-trick/tree/master